valkyrie.drush.inc

```diff
@@ -121,7 +121,11 @@ function valkyrie_drush_command() {
     'aliases' => array('vup'),
     'bootstrap' => DRUSH_BOOTSTRAP_DRUSH, // No bootstrap at all.
   );
-
+  $items['valkyrie-upgrade'] = array(
+    'description' => 'Upgrade global Valkyrie extension and all of the submodules.',
+    'aliases' => array('vug'),
```

Comment:
I'm not sure we need an alias. In fact, it might be a good idea to force someone to type it out, since it is potentially destructive.